src/utils/gen_util.rs

```diff
@@ -92,30 +92,45 @@ pub fn get_decimal_prefix(quantity: u64, unit: &str) -> (f64, String) {
     }
 }
 
-/// Gotta get partial ordering?  No problem, here's something to deal with it~
-///
-/// Note that https://github.com/reem/rust-ordered-float exists, maybe move to it one day?  IDK.
-pub fn get_ordering<T: std::cmp::PartialOrd>(
-    a_val: T, b_val: T, reverse_order: bool,
-) -> std::cmp::Ordering {
-    match a_val.partial_cmp(&b_val) {
-        Some(x) => match x {
-            Ordering::Greater => {
-                if reverse_order {
-                    std::cmp::Ordering::Less
-                } else {
-                    std::cmp::Ordering::Greater
-                }
-            }
-            Ordering::Less => {
-                if reverse_order {
-                    std::cmp::Ordering::Greater
-                } else {
-                    std::cmp::Ordering::Less
-                }
-            }
-            Ordering::Equal => Ordering::Equal,
-        },
-        None => Ordering::Equal,
```

Comment:
Removed this in favour of using `sort_partial_fn` for simpler conditional selection, or directly using `partial_ordering`/`partial_ordering_rev`.